apps/kg-explorer/src/app/components/filter-menu/filter-menu-overlay/filter-menu-overlay.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { FormControl } from '@angular/forms';
+import { render } from '@testing-library/angular';
+
+import { FilterOption } from '../../../utils/utils';
+import { FilterMenuOverlayComponent } from './filter-menu-overlay.component';
+
+describe('FilterMenuOverlayComponent', () => {
+  it('should render', async () => {
+    const promise = render(FilterMenuOverlayComponent, {
+      inputs: {
+        currentFilters: ['filter1', 'filter2'],
+        filterOptionCategory: {
+          label: 'test-category',
+        },
+        form: new FormControl<FilterOption[] | null>(null),
+      },
+    });
+    await expect(promise).resolves.toBeTruthy();
+  });
+
+  it('should select an option', async () => {
+    const { fixture } = await render(FilterMenuOverlayComponent, {
+      inputs: {
+        currentFilters: ['filter1', 'filter2'],
+        filterOptionCategory: {
+          label: 'test-category',
+          options: [{ id: 'option1', label: 'Option 1', count: 10 }],
+        },
+        form: new FormControl<FilterOption[] | null>(null),
+      },
+    });
+    const component = fixture.componentInstance;
+    component.selectOption({ id: 'option1', label: 'Option 1', count: 10 });
```

Comment:
You should be triggering this selection by emiting user events instead of programmatically accessing the methods. You can also use test [harnesses](https://material.angular.dev/components/select/api#MatSelectHarness) to trigger the events